shopware/app-bundle/1.0/config/packages/shopware_app.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('shopware_app', [
+        'shop_class' => 'App\Entity\Shop',
```

Comment:
need to check how we can maybe detect this and use:

```diff
-        'shop_class' => 'App\Entity\Shop',
+        'shop_class' => \App\Entity\Shop::class,
```

@TomasVotruba do you maybe have an idea for this?

Reply:
Hi Alex, this rule might help :) 
https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#stringclassnametoclassconstantrector